CHANGELOG.md

```diff
@@ -4,30 +4,36 @@ All notable changes to this project will be documented in this file.
 
 ---
 
-## [1.0.6] - 2021-06-04
+## [1.0.7] - 2021-06-04
 
 ### Added
 
 - Tinovi - PM-IO-5-SM
 
-## [1.0.5] - 2021-06-01
+## [1.0.6] - 2021-06-01
 
 ### Added
 
 - Comtac - CM-4
 
-## [1.0.4] - 2021-05-20
+## [1.0.5] - 2021-05-20
 
 ### Added
 
 - Dragino - LSE01
 
-## [1.0.3] - 2021-05-19
+## [1.0.4] - 2021-05-19
 
 ### Added
 
 - Digitalmatter - Oyster
 
+## [1.0.3] - 2021-05-10
```

Comment:
this should be the latest version instead of introducing an in-between version..

Reply:
@J2thatsme please revert changing these below